README.md

```diff
@@ -3,38 +3,55 @@
 [![GoReportCard](http://goreportcard.com/badge/elastic/beats)](http://goreportcard.com/report/elastic/beats)
 [![codecov.io](https://codecov.io/github/elastic/beats/coverage.svg?branch=master)](https://codecov.io/github/elastic/beats?branch=master)
 
-# Beats - Lightweight shippers for Elasticsearch & Logstash
+# Beats - The Lightweight Shippers of the Elastic Stack
 
-The [Beats](https://www.elastic.co/products/beats) are lightweight processes,
-written in Go, that you install on your servers to capture all sorts of
-operational data like logs, operating system metrics or network packet data,
-and to send it to Elasticsearch, either directly or via Logstash, so it can be
-visualized with Kibana.
+The [Beats](https://www.elastic.co/products/beats) are lightweight data
```

Comment:
This sentence has too much going on (it always did, really). I think it needs to be broken up a bit. How about changing it to this:

The Beats are lightweight data shippers, written in Go, that you install on your servers to capture all sorts of operational data (think of logs, metrics, or network packet data). The Beats send the operational data to Elasticsearch, either directly or via Logstash, so it can be visualized with Kibana.  

By "lightweight", we mean that Beats have a small installation footprint, use limited system resources, and have no runtime dependencies.